utilities/logger/src/LoggerInterface.ts

```diff
@@ -0,0 +1,9 @@
+interface LoggerInterface {
+  init(logger: any, options: any): void;
+  debug(message: string, data: any, source: string, tags: string[]): void;
+  error(message: string, data: any, source: string, tags: string[]): void;
+  info(message: string, data: any, source: string, tags: string[]): void;
+  warn(message: string, data: any, source: string, tags: string[]): void;
```

Comment:
Really strange format to me. I will start with functional tests first drafting example use cases. 
Once you will do it you will see actual problems with current interface :)
Do not want to spoil that :D